docs/data/material/components/text-fields/MultilineTextFields.tsx

```diff
@@ -24,8 +18,7 @@ export default function MultilineTextFields() {
           label="Multiline"
           multiline
           maxRows={4}
-          value={value}
-          onChange={handleChange}
+          defaultValue="Controlled"
```

Comment:
so should remove the `defaultValue` from all the input fields?

Reply:
In cases where it's "Controlled" now, yes.

Reply:
are we again having some confusion about"controlled" and "uncontrolled" by any chance? because I think I made all of the fields "uncontrolled" as per our discussion

Reply:
Yes, that's correct; all the fields are uncontrolled. That's why having a defaultValue="Controlled" is confusing and I'm asking you to remove the `defaultValue` prop where its value is "Controlled" (literally).